tutorials/CMakeLists.txt

```diff
@@ -44,9 +44,9 @@ configure_file(${CMAKE_CURRENT_SOURCE_DIR}/CTestCustom.cmake ${CMAKE_CURRENT_BIN
 #   tutorials directory which is copied to the build area.
 #-------------------------------------------------------------------
 file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/rootlogon.C "{
-  // Needed by ACLiC to use the current dicrectory for scratch area
+  // Needed by ACLiC to use the current directory for scratch area
   gSystem->SetBuildDir(\".\", kTRUE);
-  gROOT->SetWebDisplay(\"batch\");
+  gROOT->SetWebDisplay(\"batch7\");
```

Comment:
Does this mean that users trying out the tutorial will not see anything? That seems counterproductive.

Reply:
It is settings for tutorials macros which runs with `ctest` - this is content of rootlogon.C file in `runtutorials/` subfolder

Reply:
Thanks, then this is another occurrence of https://github.com/root-project/root/pull/11836#discussion_r1040582013 :-)